metricshub-web/react/src/utils/downloadAllConfigs.js

```diff
@@ -0,0 +1,31 @@
+import JSZip from "jszip";
```

Comment:
All utility functions that perform API calls should be moved to the `src/services` package. Create a new file named `downloadService.js` under `src/services` and move all download related functions into it.